<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colosseum</class>
 <widget class="QMainWindow" name="colosseum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputsLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inputs</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeDispensed_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume Dispensed</string>
    </property>
   </widget>
   <widget class="QLabel" name="setting_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QLabel" name="value_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="units_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Units</string>
    </property>
   </widget>
   <widget class="QComboBox" name="setting1_combo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Total time</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Total volume</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="setting2_combo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Volume per fraction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Number of fractions</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="value1_line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="value2_line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="unit1_combo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="unit2_combo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="flowrate_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow Rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="flowrate_line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="flowunit_combo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>uL/sec</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>uL/min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>uL/hr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mL/sec</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mL/min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mL/hr</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="tubeunit_combo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0.5mL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5mL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.0mL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5.0mL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="tubesize_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tube Size</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="voldispensed_disp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>147</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="voldispensed_unit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mL</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeElapsed_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>124</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Elapsed</string>
    </property>
   </widget>
   <widget class="QLabel" name="tubeNumber_label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tube Number</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="timeElapsed_disp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tubeIteration_disp">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="setting3_combo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Total time</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Total volume</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="setting4_combo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Volume per fraction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Number of fractions</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="value3_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="value4_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QComboBox" name="unit3_combo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="unit4_combo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseresumeButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>200</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause/Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
